### PR TITLE
feat: modify requests to align with apis

### PR DESCRIPTION
**Pull request by benox3**

### CODE REVIEW

File: apps/api/src/server.ts [hunk 1]
```diff
@@ -61,104 +60,88 @@ export const createServer = () => {
...
-            },
-          },
-        );
 
-        // Make a request to get claims from internal APIs
-        const claims = MOCKED_CLAIMS;
-
-        // Make a request to get the native user id from internal APIs
+        // Integrate [Update or Create a User](https://docs.getportabl.com/api-ref#update-or-create-a-user) endpoint
+        //  - You shall enable data syncronization for a user by passing the following request body:
```
Comment (bao-portabl) [inline apps/api/src/server.ts:86]: typo: synchronization

File: apps/api/src/server.ts [hunk 2]
```diff
@@ -61,104 +60,88 @@ export const createServer = () => {
...
+        //    - `{ settings: { isSyncOn: true } }`
+        //  - If user is not found by (linked to) your Portabl account yet, it will create user in (link to) your Portabl account;
```
Comment (bao-portabl) [inline apps/api/src/server.ts:88]: Can we remove the (linked to) and (link to), I think it makes sense without the additional context. Had to re-read that a few times to to understand the parens

File: apps/api/src/server.ts [hunk 3]
```diff
@@ -61,104 +60,88 @@ export const createServer = () => {
...
+        //  - Otherwise, if user is found, it will update user settings and set data syncronization flag to true;
```
Comment (bao-portabl) [inline apps/api/src/server.ts:89]: typo: synchronization

File: apps/api/src/server.ts [hunk 4]
```diff
@@ -61,104 +60,88 @@ export const createServer = () => {
...
+        // - Patches claims of a user;
+        // - By default, patching user claims will start the data synchronization process;
+        // - To disable this behaviour and control when data synchronization should start you can provide "noAutoSync=false" in your query params;
+        axios
+          .patch(
+            `${baseUrl}/provider/users/${userId}/claims`,
+            { claims },
+            { headers: { authorization: `Bearer ${ACCESS_TOKEN}` } },
+          )
+          .then((result) =>
```
[the remark this answers is not in the document]
Comment (benox3) [reply apps/api/src/server.ts:122]: Yea agreed - let me polish up the server.

File: apps/api/src/server.ts (hunk 2 above)
Comment (benox3) [reply apps/api/src/server.ts:88]: Agreed -- will simplify this section entirely.

**Pull request merged**